doc/release/1.2.0-notes.rst

```diff
@@ -213,17 +213,17 @@ The functions ``hyp2f0``, ``hyp1f2`` and ``hyp3f0`` in ``scipy.special`` have
 been deprecated.
 
 
-Backwards incompatible changes
+Backwards-incompatible changes
 ==============================
 
 LAPACK version 3.4.0 or later is now required. Building with
 Apple Accelerate is no longer supported.
 
 The function ``scipy.linalg.subspace_angles(A, B)`` now gives correct
 results for all angles. Before this, the function only returned
-correct values for those angles which were greater than pi/4.
+correct values for those angles which were greater than π/4.
```

Comment:
Thanks for reviewing the changes! I've given it some thought and noticed the following things: pi as 'pi' appears sometimes but so does pi as the symbol AND pi as the ':math:' symbol. That's why I thought it was safe to paste the symbol (because it was already elsewhere in the docs). I'm now following the convention of using ':math:' pi symbol everywhere. It is consistent with almost every tutorial / file. As for the release note 1.2.0, I won't change it anymore. But it was good to experiment with some conventions in the release notes to see how they turn out.

Reply:
`:math:some_latex_expression` renders math and displays it with LaTeX (for pdf) or MathJax (for html). Nice for rendered docs, but less readable in a terminal. So we try to keep the use of `:math:` to tutorials or Notes sections of docstrings.

Reply:
Thanks, I'll definitely bear that in mind.